Documentation/direct-mount.md

```diff
@@ -0,0 +1,60 @@
+#rename this file as direct-tools.yaml to create a toolbox pod out of it
```

Comment:
This file should be in the same directory as toolbox.yaml and please name it direct-mount.yaml